config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,16 @@
+require 'okcomputer'
+
+OkComputer.check_in_parallel = true
+
+##
+# REQUIRED checks
+
+# Simple echo of the VERSION file
+class VersionCheck < OkComputer::AppVersionCheck
+  def version
+    File.read(Rails.root.join('VERSION')).chomp
+  rescue Errno::ENOENT
+    raise UnknownRevision
+  end
+end
+OkComputer::Registry.register 'version', VersionCheck.new
```

Comment:
FYI:  I have a PR into okcomputer that updates their version check to Capistrano 3:  https://github.com/sportngin/okcomputer/pull/112.  No idea if they're ever going to take the PR but it would be another way to get this information, without having to remember to update the VERSION file.

Another possibility would be to name the file REVISION and use the existing okcomputer check.